test/types/schema.test-d.ts

```diff
@@ -1,5 +1,6 @@
 import { expectAssignable, expectError } from 'tsd'
 import fastify, { FastifyInstance } from '../../fastify'
+import * as Ajv from 'ajv'
```

Comment:
Namespace should not be callable/newable. Use `import Ajv = require('ajv')` to make it work in all scenarios.

Reply:
Sure